clock.go

```diff
@@ -231,21 +232,42 @@ func (a clockTimers) Less(i, j int) bool { return a[i].Next().Before(a[j].Next()
 // Timer represents a single event.
 // The current time will be sent on C, unless the timer was created by AfterFunc.
 type Timer struct {
-	C     <-chan time.Time
-	c     chan time.Time
-	timer *time.Timer // realtime impl, if set
-	next  time.Time   // next tick time
-	mock  *Mock       // mock clock, if set
-	fn    func()      // AfterFunc function, if set
+	C          <-chan time.Time
+	c          chan time.Time
+	timer      *time.Timer // realtime impl, if set
+	next       time.Time   // next tick time
+	mock       *Mock       // mock clock, if set
+	fn         func()      // AfterFunc function, if set
+	registered bool        // True if active, false if stopped
```

Comment:
Can you rename this to `stopped` and flip the `true`/`false` values? e.g. `// True if stopped, false if active`.